var/spack/repos/builtin/packages/ucx/package.py

```diff
@@ -44,4 +51,9 @@ def configure_args(self):
             config_args.append('--enable-mt')
         else:
             config_args.append('--disable-mt')
+        if '+cuda' in spec:
+            config_args.append('--with-cuda={0}'.format(spec['cuda'].prefix))
+        if '+gdrcopy' in spec:
+            config_args.append('--with-gdrcopy')
```

Comment:
Can these be disabled with `--without`?

Reply:
There are actually no 'without' options for these flags, and specifying no 'with' options in effect disable them